tests/Functional/Fixtures/Symfony/AppCache.php

```diff
@@ -25,4 +29,14 @@ public function handle(Request $request, $type = HttpKernelInterface::MASTER_REQ
 
         return $response;
     }
+
+    /**
+     * Made public to allow event subscribers to do refresh operations.
+     *
+     * {@inheritDoc}
+     */
+    public function fetch(Request $request, $catch = false)
```

Comment:
This needs to be overridden here because of trait [precedence](http://php.net/manual/en/language.oop5.traits.php#language.oop5.traits.precedence).